### DISCUSSION TITLE
public schools students being accepted

### DISCUSSION THREAD

**Discussion by awkkih** [community/community#39895] [GitHub Education]
hello everyone, im here to ask if public schools students on Brazil have any chance of getting into GitHub Education Developer Pack
i already submitted like 3 times but never got accepted. do you think if i send a pic of my ID its going to work?

thanks in advance

**Comment by radar-blip** [comment]
According to the article here: https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/
a `dated student ID` will help, yes.

**Comment by samus-aran** [comment]
Hi @Akkihiko  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

P.S A Student ID is much preferred over other types of evidence for most institutes ⭐

**Reply by awkkih** [reply]
hey, thanks for the response
@samus-aran, it just says the image could be modified in some way, even though it isnt
but i'll try that, thanks!
